src/frontend-admin/src/services/AxiosService/AxiosService.ts

```diff
@@ -0,0 +1,135 @@
+import axios, {AxiosInstance} from 'axios'
+//import qs from 'qs' ??
+//import buildQuery from 'odata-query' ??
+import decode from 'jwt-decode'
+
+export interface IUserInfo {
+    name: string
+    token: string
+    refreshToken: string
+    isAdmin: boolean
+}
+
+//TODO: find out actual baseURL && secret or import from somewhere?
+const URL= "";
```

Comment:
This one is going to come from the environment variables, which would look something like `process.env.REACT_APP_API_URL`. This is how the frontend will be reusable by other interested parties - they can plug in their own api url in in their own env variables. See https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables for more details